.eslintrc.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
   globals: {
     contract: false,
     assert: false,
+    web3: true,
+    artifacts: true,
```

Comment:
These should be scoped to the files they are actually globals in. which as far as I know is just deploy scripts and 0_setup_tests.js

Reply:
also Artifacts.js and a couple others I believe. I notice I am adding it often though, and not sure why it should be scoped locally